### ISSUE TITLE
Error: Could not find or load main class uk.ac.ebi.ena.downloader.gui.Main

### ISSUE THREAD

**Issue by minisciencegirl** [enasequence/ena-ftp-downloader#5]
I am having the same issue as the previously closed issue.

I have java-8-oracle installed and also did `sudo apt-get install openjfx`, but am receiving this same error.

**Comment by suranjayathilaka** [comment]
Hi Amy, 

With Oracle Java 8 you should not need openjfx at all. Could you tell me which Operating System you're running, the java version (java -version) , and the full command you used to run the app?

Thanks!

**Comment by minisciencegirl** [comment]
I am running Ubuntu 16.04 LTS with openjdk version "1.8.0-121"

The command I am using to run the app is: `java -jar ena-file-downloader-v1.2.1.jar`

Thanks!

**Comment by tobsecret** [comment]
Having this same issue. 
```
openjdk version "1.8.0_171"
OpenJDK Runtime Environment (build 1.8.0_171-b10)
OpenJDK 64-Bit Server VM (build 25.171-b10, mixed mode)
```

**Comment by suranjayathilaka** [comment]
With the newest release we've made available ena-file-downloader-v1.2.2-with-runtime.zip which contains the Oracle runtime, and also self-contained .dep and .rpm packages.
Please try using one of those and let me know.
Thank you!
